## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 438538`
Commit: `4c5e4c6743125b6cacedf7d16d3bbf28fc3d7958`

---
### `x86_64-linux`
<details>
  <summary>:x: 3 packages failed to build:</summary>
  <ul>
    <li>vkd3d</li>
    <li>vkd3d.dev</li>
    <li>vkd3d.lib</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-linux`</summary>
<details>
<summary>vkd3d</summary>
<pre>checking for sys/stat.h... yes
checking for sys/types.h... yes
checking for unistd.h... yes
checking for wchar.h... yes
checking for minix/config.h... no
checking whether it is safe to define __EXTENSIONS__... yes
checking whether _XOPEN_SOURCE should be defined... no
checking the archiver (ar) interface... ar
checking for gcc... (cached) gcc
checking whether the compiler supports GNU C... (cached) yes
checking whether gcc accepts -g... (cached) yes
checking for gcc option to enable C11 features... (cached) none needed
checking whether gcc understands -c and -o together... (cached) yes
checking for a sed that does not truncate output... /nix/store/i74283mw5bncn16i0zbz0lvvq4sn0q87-gnused-4.9/bin/sed
checking for a race-free mkdir -p... /nix/store/xbp2j3z0lhizr5vvzff4dgdcxgs8i2w7-coreutils-9.7/bin/mkdir -p
checking for gawk... gawk
checking for widl... widl
checking whether widl version >= 3.21... yes (10.0)
checking for perl... no
configure: error: no suitable perl found. Please install the 'perl-base' package.</pre>
</details>
</details>

--- a/pkgs/by-name/vk/vkd3d/package.nix
+++ b/pkgs/by-name/vk/vkd3d/package.nix
@@ -4,6 +4,7 @@
   bison,
   fetchFromGitLab,
   flex,
+  perlPackages,
   pkg-config,
   spirv-headers,
   stdenv,
@@ -14,14 +15,14 @@
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "vkd3d";
-  version = "1.15";
+  version = "1.17";
 
   src = fetchFromGitLab {
     domain = "gitlab.winehq.org";
     owner = "wine";
     repo = "vkd3d";
-    rev = "vkd3d-${finalAttrs.version}";
-    hash = "sha256-EzXsYi9wC+JdMYE77cIeO2lV+fY4ViQM3+KcAJFv9tU=";
+    tag = "vkd3d-${finalAttrs.version}";
+    hash = "sha256-jxQ9L1GL4j3P5/nb79qAXQp8/IStOWmiK/vvbFxeg1k=";
   };
 
   outputs = [
@@ -34,6 +35,8 @@ stdenv.mkDerivation (finalAttrs: {
     autoreconfHook
     bison
     flex
+    perlPackages.perl
+    perlPackages.JSON
     pkg-config
     wine
   ];

--- a/pkgs/by-name/vk/vkd3d/package.nix
+++ b/pkgs/by-name/vk/vkd3d/package.nix
@@ -64,7 +64,7 @@ stdenv.mkDerivation (finalAttrs: {
     '';
     license = with lib.licenses; [ lgpl21Plus ];
     mainProgram = "vkd3d-compiler";
-    maintainers = with lib.maintainers; [ ];
+    maintainers = with lib.maintainers; [ liberodark ];
     inherit (wine.meta) platforms;
   };
 })

